bsc-plugin/src/lib/rooibos/CodeCoverageProcessor.spec.ts

```diff
@@ -333,7 +333,7 @@ describe('RooibosPlugin', () => {
                 expect(a).to.equal(b);
             });
 
-            it('correctly transpiles some statements', async () => {
+            it.skip('correctly transpiles some statements', async () => {
```

Comment:
Investigating a fix for this failing test with @TwitchBronBron

Reply:
```suggestion
            it('correctly transpiles some statements', async () => {
```

Rolled back to brighterscript `0.67.4` till we can solve this issue caused by changes in `0.68`